TramsDataApi/Gateways/IAcademyTransferProjectGateway.cs

```diff
@@ -1,16 +1,29 @@
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using TramsDataApi.DatabaseModels;
 
 namespace TramsDataApi.Gateways
 {
-    public interface IAcademyTransferProjectGateway
-    {
-        AcademyTransferProjects SaveAcademyTransferProject(AcademyTransferProjects project);
-        AcademyTransferProjects GetAcademyTransferProjectByUrn(int urn);
-        AcademyTransferProjects UpdateAcademyTransferProject(AcademyTransferProjects project);
-        IList<AcademyTransferProjects> IndexAcademyTransferProjects(int page);
-        IList<AcademyTransferProjects> GetAcademyTransferProjects();
+   public interface IAcademyTransferProjectGateway
+   {
+      AcademyTransferProjects SaveAcademyTransferProject(AcademyTransferProjects project);
+      AcademyTransferProjects GetAcademyTransferProjectByUrn(int urn);
+      AcademyTransferProjects UpdateAcademyTransferProject(AcademyTransferProjects project);
 
+      /// <summary>
+      ///    Retrieves the specified page of transfer projects and the total number of projects in the database
+      /// </summary>
+      /// <param name="page">the page of results to return</param>
+      /// <returns>
+      ///    A <see cref="Tuple{IList, Int32}" /> containing the requested page of results and the total number of matching
+      ///    entries in the database
+      /// </returns>
+      /// <remarks>
+      ///    <para>Page size is hard-coded to ten</para>
+      ///    <para><paramref name="page" /> is 1-based, not zero-based (Page 1 is the first entry, not page zero)</para>
+      /// </remarks>
+      Tuple<IList<AcademyTransferProjects>, int> IndexAcademyTransferProjects(int page);
```

Comment:
This can be shortened to (IList<AcademyTransferProjects>, int)

Reply:
That would then give no hints about the return type to users of the interface, though.

Reply:
Oh fair enough. I thought it was syntactic sugar but meant the same thing. Makes sense then.